src/Stryker.Core/Stryker.Core/Initialisation/InitialTestProcess.cs

```diff
@@ -31,6 +31,10 @@ public InitialTestProcess()
         public int InitialTest(ITestRunner testRunner)
         {
             var message = testRunner.DiscoverNumberOfTests() is var total && total == -1 ? "Unable to detect" : $"{total}";
+            if (total == 0)
```

Comment:
This is already gracefully handled by the mutationtestprocess here

 https://github.com/stryker-mutator/stryker-net/blob/cfc1e1cb3f0f9ef17bfab1d6a0a6e15cc7b01e16/src/Stryker.Core/Stryker.Core/MutationTest/MutationTestProcess.cs#L178

It will log a warning that none of the mutants were covered by a test and it will create the report with all created mutations. We do not need to throw on no tests found.